api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1875,7 +1874,7 @@ public void makeBucket(String bucketName, String region)
       configString = config.toString();
     }
 
-    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, 0);
+    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, configString.length());
```

Comment:
> it is largely configString.getBytes("UTF-8").length()

yes, I meant that if this is about bucket creation, we won't have any unicode character inside the string, am I correct?